CircularBuffer.h

```diff
@@ -5,6 +5,10 @@ On 18 April 2014, the simplified version on the Wikipedia page for power of 2 si
 doesn't work - cbIsEmpty() returns true whether the buffer is full or empty.
 */
 
+#define MOZZI_BUFFER_SIZE 256 // do not expect to change and it to work.
```

Comment:
I did that exactly because one of my long term idea would be to make the mozzi buffers adjustable in sizes. As I think this is the only thing that would require to be adapted in Mozzi for different buffer sizes (except the buffer themselves of course), I was anticipating a bit and not wanted to bury that to far down in the code.